src/lib/snarky_bn382_backend/pairing_based.ml

```diff
@@ -83,7 +103,7 @@ module Keypair = struct
   let set_urs_info, load_urs =
     let urs_info = Set_once.create () in
     let urs = ref None in
-    let set_urs_info ?(degree = 2 * 786_433) path =
+    let set_urs_info ?(degree = 3_11) path =
```

Comment:
Is this number large enough

Reply:
Pairing backend is only used for a small circuit in the test. Have chosen small value for saving test time